### DISCUSSION TITLE
Adding servers for LAN while not on LAN network

### DISCUSSION THREAD

**Discussion by jaselee0905** [PhlexPlexico/G5API#278] [Help]
I have the site running on my server at home, I plan to use this for a LAN and would like to add the servers to it but I'm struggling to figure out the best way to do this, I thought I might be able to reverse proxy the lan servers and add them to the site but that didn't work I tried to use a vpn but I haven't had much luck with that either

**Comment by PhlexPlexico** [comment]
Hey there, 

I don't think there would be a way to reverse proxy a game server out to the API. I think what you would need to do is either setup your firewall on LAN to accept outside connections for specific hosts/ports, and set the game servers to be `sv_lan 0`.

Another option would be to use an SBC like a raspberry pi and bring that to your LAN and operate the API from there, so you can make adjustments to the API (and any adjustments to the hosts file on said SBC and game servers) to make it more LAN friendly (i.e. giving hostnames for the API and disabling SSL). I'm not sure if there's a possible way to use it for LAN servers unless you open up the game servers themselves to external connections.

Thanks!
